database/migrations/2014_10_12_000000_create_users_table.php

```diff
@@ -20,10 +20,10 @@ public function up()
             $table->string('firstname');
             $table->string('lastname');
             $table->string('code');
-            // $table->timestamp('email_verified_at')->nullable();
-            // $table->string('password');
-            // $table->rememberToken();
-            // $table->timestamps();
+            $table->timestamp('email_verified_at')->nullable();
```

Comment:
Hey @bekeleh, the users table doesn't need these fields, I feel they could just be removed